models/demos/metal_BERT_large_11/tt/bert_encoder.py

```diff
@@ -27,37 +26,37 @@ def __init__(self, config, encoder_idx, state_dict, device, model_config, tt_cac
         layer_name = f"bert.encoder.layer.{encoder_idx}.output"
 
         if tt_cache_path is not None:
-            self.attention_output_weight = tt_lib.tensor.load_tensor(
+            self.attention_output_weight = ttnn.experimental.tensor.load_tensor(
```

Comment:
Don't we have `ttnn.load_tensor` that can be used? Or is there some difference preventing this?

Reply:
We are working based on the inputs we receive from the TT team as requested https://tenstorrent.slack.com/archives/C079KU1LNC9/p1723214250435539 
And load_tensor was found in ttnn/cpp/ttnn/deprecated/tt_lib/csrc/tt_lib_bindings_tensor.cpp as well, so both apis exist, not sure ttnn.load_tensor supports all functionalities, same as tt_lib one.